src/main/java/com/autotune/operator/KruizeDeploymentInfo.java

```diff
@@ -66,14 +66,12 @@ public class KruizeDeploymentInfo {
     public static Boolean settings_save_to_db;
     public static String em_only_mode;
     public static Integer bulk_update_results_limit = 100;
-
     public static int generate_recommendations_date_range_limit_in_days = 15;
-
+    public static String delete_partition_threshold_in_days;
```

Comment:
Can you please update the docs on how to use this?

Reply:
Has this been tested?

Reply:
yes on [EE](https://console-openshift-console.apps.c-rh-c-eph.8p0c.p1.openshiftapps.com/k8s/cluster/projects/ephemeral-ujnc03) and created [MR](https://github.com/RedHatInsights/ros-ocp-backend/pull/163) too

Reply:
> Can you please update the docs on how to use this?

done

Reply:
Please include the test into our kruize tests to validate both default and custom threshold value

Reply:
@chandrams Could you kindly provide a reference test suite for verifying default and custom values set via environment variables? Or, is a new test suite needed for this purpose? Also, how are other params unrelated to this PR being tested?

Reply:
We do not have any tests related to environment values.

Reply:
Chandrakal, appreciate it. This appears to be a significant change, requiring a framework for assessment. Should we contemplate creating a distinct PR to address the broader scope?